IFC4x3/Templates/Object Connectivity/Spatial Structure/Spatial Interference/Spatial Interference With Zones/DocTemplateDefinition.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocTemplateDefinition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Spatial Interference With Zones" UniqueId="a4b43422-04a3-45c0-aa4b-725e391ec67b" Type="IfcSpatialElement">
+	<Rules>
+		<DocModelRuleAttribute Name="InterferesElements" Identification="InterferesSpatialElements">
+			<Rules>
+				<DocModelRuleEntity Name="IfcRelInterferesElements">
+					<Rules>
+						<DocModelRuleAttribute Name="RelatedElement" Identification="RelatedSpatialElement">
+							<Rules>
+								<DocModelRuleEntity Name="IfcSpatialElement">
+									<Rules>
+										<DocModelRuleAttribute Name="Name" Identification="RelatedSpatialElementName">
+											<Rules>
+												<DocModelRuleEntity Name="IfcLabel" />
+											</Rules>
+										</DocModelRuleAttribute>
+									</Rules>
+								</DocModelRuleEntity>
+							</Rules>
+						</DocModelRuleAttribute>
+						<DocModelRuleAttribute Name="InterferenceType" Identification="InterferenceType">
+							<Rules>
+								<DocModelRuleEntity Name="IfcIdentifier" />
+							</Rules>
+						</DocModelRuleAttribute>
+						<DocModelRuleAttribute Name="InterferenceSpace" Identification="InteferenceZone">
+							<Rules>
+								<DocModelRuleEntity Name="IfcSpatialZone">
+									<Rules>
+										<DocModelRuleAttribute Name="PredefinedType" Identification="SpatialZoneType">
+											<Rules>
+												<DocModelRuleEntity id="IfcSpatialZoneTypeEnum_0000000000000000000000" Name="IfcSpatialZoneTypeEnum" Description="*">
+													<Rules>
+														<DocModelRuleConstraint>
+															<Expression xsi:type="DocOpStatement" Operation="compareequal">
+																<Reference xsi:type="DocOpReference">
+																	<EntityRule xsi:type="DocModelRuleEntity" xsi:nil="true" href="IfcSpatialZoneTypeEnum_0000000000000000000000" />
+																</Reference>
+																<Value xsi:type="DocOpLiteral" Operation="loadstring" Literal="INTERFERENCE" />
```

Comment:
Just a check out of ignorance that this is the correct syntax, and that it should not be ".INTERFERENCE." as in the STEP format?

Reply:
@SergejMuhic, can you double check my syntax on this.